include/envoy/upstream/upstream.h

```diff
@@ -591,8 +591,8 @@ class PrioritySet {
   GAUGE(membership_healthy, NeverImport)                                                           \
   GAUGE(membership_total, NeverImport)                                                             \
   GAUGE(upstream_cx_active, Accumulate)                                                            \
-  GAUGE(upstream_cx_rx_bytes_buffered, Accumulate)                                                 \
-  GAUGE(upstream_cx_tx_bytes_buffered, Accumulate)                                                 \
+  GAUGE(upstream_cx_rx_bytes_buffered, NeverImport)                                                \
+  GAUGE(upstream_cx_tx_bytes_buffered, NeverImport)                                                \
```

Comment:
I feel like these should be accumulate. Do they use set just for convenience? It seems like the code could be changed to not use set?